monai/networks/layers/spatial_transforms.py

```diff
@@ -439,7 +439,7 @@ def __init__(
         normalized: bool = False,
         mode: str = GridSampleMode.BILINEAR,
         padding_mode: str = GridSamplePadMode.ZEROS,
-        align_corners: bool = False,
+        align_corners: bool = True,
```

Comment:
May I know why change the default value of `align_corners`?

Thanks.

Reply:
there's a bug in the current dev implementation, the default behaviour of `AffineTransform()` is actually the same as `AffineTransform(align_corners=True)`